src/components/Sum/Sum.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a, b }) => (
```
You can leave them in one line if there are < 3 props and the line is not too long